src/main/java/forest/detector/dao/repository/TicketRepository.java

```diff
@@ -0,0 +1,27 @@
+package forest.detector.dao.repository;
+
+import forest.detector.dao.entity.Ticket;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class TicketRepository {
+
+    public void addTicket(Ticket ticket) {
+        DataSource dataSource = new DataSource();
+        String query = "INSERT INTO forest.tickets(number, region, forest_user, start_date, finish_date, " +
```

Comment:
use PreparedStatement for SQL queries
https://docs.oracle.com/javase/tutorial/jdbc/basics/prepared.html